### PR TITLE
Upgrading latest version of VSCode 3.6.2 plugin

### PR DESCRIPTION
**Pull request by AbhiAuto**
### What does this PR do?
This PR is for latest VS Code plugin for Eclipse Che

### What issues does this PR fix or reference?

It will help the testing community to leverage the Eclipse Che with VS Code for their test automation development and execution using TestNG and JUnit. As Java "https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-test" is not compatible with the existing version of VS Code currently in the plugin registry repository.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: v3/plugins/cdr/code-server/v3.6.2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+apiVersion: v2
+publisher: cdr
+name: code-server
+version: v3.6.2
```
Comment (ericwill) [inline v3/plugins/cdr/code-server/v3.6.2/meta.yaml:4]: the `v` isn't needed here, only `3.6.2`

File: v3/plugins/cdr/code-server/latest.txt [hunk 2]
```diff
@@ -1 +1 @@
-2.1650-vsc1.39.2
\ No newline at end of file
+v3.6.2
```
Comment (ericwill) [inline v3/plugins/cdr/code-server/latest.txt:1]: the `v` isn't needed here, only `3.6.2`

File: v3/plugins/cdr/code-server/v3.6.2/meta.yaml (hunk 1 above)
Comment (AbhiAuto) [reply v3/plugins/cdr/code-server/v3.6.2/meta.yaml]: Updated !

File: v3/plugins/cdr/code-server/latest.txt (hunk 2 above)
Comment (AbhiAuto) [reply v3/plugins/cdr/code-server/latest.txt]: Updated !

**Review by ericwill — APPROVED** [review]

**Pull request merged**